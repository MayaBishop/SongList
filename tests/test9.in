m 100
i Mamma Mia;Abba
i The Winner Takes it All;Abba
i Dancing Queen;Abba
i Honey Honey;Abba
i We Will Rock You;Queen
i We Will Rock You;Queen2
i Venus;Bananarama
i Crocodile Rock;Elton John
i All that she wants;Ace of Base
i Night and Day;Frank Sinatra
i Downtown;Petula Clark
p 3
p 5
p 6
