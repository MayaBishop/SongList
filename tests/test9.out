success
success
success
success
success
success
success
success
success
success
success
success
played 3 Dancing Queen;Abba
played 5 We Will Rock You;Queen
played 6 We Will Rock You;Queen2
